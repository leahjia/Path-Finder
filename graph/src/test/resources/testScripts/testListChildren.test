# DesignMap listing children nodes testing file.
# Tests that a list of destination nodes from a given node can be computed

# Create a graph
CreateGraph map1

# Add a pair of edges
AddEdge map1 n1 n2 e2
AddEdge map1 n1 n2 e1
AddEdge map1 n2 n1 e4
AddEdge map1 n2 n1 e3

# Finds the children of node n1
ListChildren map1 n1

# Finds the children of node n2
ListChildren map1 n2