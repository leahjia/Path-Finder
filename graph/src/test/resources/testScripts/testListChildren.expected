# DesignMap listing children nodes testing file.
# Tests that a list of destination nodes from a given node can be computed

# Create a graph
created graph map1

# Add a pair of edges
added edge e2 from n1 to n2 in map1
added edge e1 from n1 to n2 in map1
added edge e4 from n2 to n1 in map1
added edge e3 from n2 to n1 in map1

# Finds the children of node n1
the children of n1 in map1 are: n2(e1 e2)

# Finds the children of node n2
the children of n2 in map1 are: n1(e3 e4)
